### DISCUSSION TITLE
pambi.tech: scam

### DISCUSSION THREAD

**Discussion by calix6729** [uBlockOrigin/uAssets#24317] [General]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
www.pambi.tech/
```

### Description

This is not an scam website, pleae remove it from the filters, there are no bots spaming the website is just a crypto community

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![GRN1RFiXoAAq75a](https://github.com/uBlockOrigin/uAssets/assets/174214863/ac0e2984-fa8f-4d0b-a039-20fd7b30cfbc)
https://gyazo.com/6bf920d5c4a03e257c4cad7e7ed297bc

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml


```
</details>

**Comment by Jeinsx** [comment]
This is not a scam site, it's a crypto community site to know news and stay tuned about their token activities.

**Comment by YuxinDesigner** [comment]
I have been looking at the website in a totally ecceptic way, and I already have a conclusion. It is impossible that it is a website to scam, as they never ask for personal data of users such as: passwords, emails, or bank details such as credit or debit cards. It is simply a website that serves as a news portal for its crypto community.

**Comment by droidh3ll** [comment]
> I have been looking at the website in a totally ecceptic way, and I already have a conclusion. It is impossible that it is a website to scam, as they never ask for personal data of users such as: passwords, emails, or bank details such as credit or debit cards. It is simply a website that serves as a news portal for its crypto community.

This website is indeed a phishing domain. The website prompts you for your wallet for unknown purposes and is **not associated with any of the brands** that appear on it.

**Comment by JobcenterTycoon** [comment]
Removed for now.

https://www.virustotal.com/gui/url/11113a8997e75c983a81297de7b71a1e9807b8e0c079a398c8b0ef743048a5f4/detection 0 VT detections.

Additional proofs are needed.

**Comment by droidh3ll** [comment]
> Removed for now.
> 
> https://www.virustotal.com/gui/url/11113a8997e75c983a81297de7b71a1e9807b8e0c079a398c8b0ef743048a5f4/detection 0 VT detections.
> 
> Additional proofs are needed.

None of the brands appearing on it are associated as seen [here](https://twitter.com/nitrolauncher/status/1805299393563869571). Users who fall into this domain are likely to lose their crypto assets.

**Comment by calix6729** [comment]
False

https://x.com/StarNestweb3/status/1806311404640837782
https://x.com/0xgen_io/status/1805892834056458249
https://x.com/GameSpace_Web3/status/1805875500013043734
https://x.com/ONTOWallet/status/1805845842227871794

Just few examples of few examples of the partners, there are many more in https://x.com/pambicoin

**Comment by droidh3ll** [comment]
> False
> 
> https://x.com/StarNestweb3/status/1806311404640837782 https://x.com/0xgen_io/status/1805892834056458249 https://x.com/GameSpace_Web3/status/1805875500013043734 https://x.com/ONTOWallet/status/1805845842227871794
> 
> Just few examples of few examples of the partners, there are many more in https://x.com/pambicoin

Those domains are unrelated to the ones shown on the website. Meaning that it's, indeed, deceptive.

**Comment by calix6729** [comment]
One of the team members already answererd to that https://x.com/FerUrdanetaDev/status/1807116198506574235 the account that you are mention is not objective

**Comment by YuxinDesigner** [comment]
I was looking at that "Needs connection to your wallet", I did it, and it really has nothing out of the ordinary, if we look at other websites that do the same thing, (Like [prom.io](https://prom.io)) they only ask people for authorization to connect the wallet, but they don't ask permissions for other things like making purchases or sales with your wallet. It says it explicitly when you connect your wallet and read the permissions they ask for.

I see that the reason you need to connect your wallet to the website, is because the organizers are working on an Airdrop for users who complete easy tasks, so that when they complete them, they release the rewards to the wallets that participated. I really don't see anything out of the ordinary with this website.

**Comment by shootlazer** [comment]
It seems a bit suspicious that most of the accounts defending this domain were created today. 
Beware.

**Comment by calix6729** [comment]
I know, I just created my account because I didn´t have one and I wanted to take out the web from that list, the other account are people from the community, you are able to search all the twitter account that are sharing the link and you will be able to see that we are real people, just click on the profiles https://x.com/search?q=pambi.tech%2Fairdrop&src=typed_query&f=top

**Comment by YuxinDesigner** [comment]
> It seems a bit suspicious that most of the accounts defending this domain were created today. Beware.

My account was not created today, I am being completely unbiased and telling what I see in detail.

**Comment by YuxinDesigner** [comment]
> > Removed for now.
> > https://www.virustotal.com/gui/url/11113a8997e75c983a81297de7b71a1e9807b8e0c079a398c8b0ef743048a5f4/detection 0 VT detections.
> > Additional proofs are needed.
> 
> None of the brands appearing on it are associated as seen [here](https://twitter.com/nitrolauncher/status/1805299393563869571). Users who fall into this domain are likely to lose their crypto assets.

Oh, I see, apparently they call this website a "Scam" for having supposed partners claiming that they have never had any relation with the crypto project. Well, at first I said "Oh, now I understand why they say it's a scam", I took longer than expected, but doing a lot of research, I found one of the developers talking about it, ([Post here](https://x.com/FerUrdanetaDev/status/1807116198506574235)) it was difficult, but finally I understood what they meant in this post made by one of the developers. 

This person made it clear, that the only thing they are looking for is to form an official and formal partnership with those companies, that's why they put them in "partnering with" and not in "our partners". I went to the official X ([Twitter](https://x.com/pambicoin)) account of the PAMBII project, and apparently they have already made many partnerships with many companies and very high level, [like this one](https://x.com/HIBTGlobal/status/1806941239167857066).

Looking at it all in the most objective way possible, they probably would have used other words for the issue of the partners, but well, this guy named Fernando Urdaneta clarified the situation, now I understand it.

**Comment by droidh3ll** [comment]
> I don't speak Spanish, so I had to translate everything, but finally I understood what they meant in this post made by one of the developers.

It's kinda funny, because a simple Google search shows that not only you do speak spanish, [Andrés Mora](https://twitter.com/YuxinDesigner), but you also seem to be one of the people involved with this site.

The brands already stated that **the use of their logos is unauthorized**, so it's a deceptive site and the use of Coin98, RhinoFi, TrustWallet, Secure3, Galxe, LiFi and LayerBank logos is misleading.

**Comment by YuxinDesigner** [comment]
According to Fernando's comments, these logos are only partners they aspire to reach, in order to achieve official partnerships. That is why it does not say "partnership with" and not "Our partners".

**Comment by JND94** [comment]
@JobcenterTycoon Please don't fall for a raid of new created accounts and a guy (Yuxinexe) who's clearly involved in the "project", see link of his twitter account above.
This is an already proven pump and dump scam and creator is going to face various collective lawsuits.
https://twitter.com/CryptoRugMunch/status/1806140521041318289
https://twitter.com/Cryptobruj/status/1806376205752643633

**Comment by Jeinsx** [comment]
Tha account linked above has desmostrated to have something personal against the creator by accussing him of horrible things by using altered wikipedia info and old news that the creator himself demostrated were fake and that he won the trials by prooving he was innocent.
https://x.com/CryptoRugMunch/status/1806390314837450911

Also there is prooves that the owner of that account "cryptorugmunch" doesn't want to seek the thruth since the creator of the token "Dalas" tried to tell him the other side of the story and give prooves of his innocence, but "CRM" didn't want to let him defense himself and instead just banned him from his tg group.
https://x.com/DalasReview/status/1806408006898266287

**Comment by mapx-** [comment]
Highly controversial => I'll convert it into discussion

**Reply by calix6729** [reply]
> Highly controversial => I'll convert it in discussion

Sorry, the coin is controversial on the spanish comunity, coul you please inform to us if the page will be on the list again? I don´t understand why it was re opened

**Reply by krystian3w** [reply]
I think it's mainly due to the amount of Emojis in the reactions.

Then, how competent is someone in detecting whether a given website is a financial pyramid that will disappear within a year and deceive hundreds of people.

**Reply by JND94** [reply]
Basically ublock maintainers got fooled by an organized raid with new created accounts to unlist their scam.
This is not questionable, it's a classic pump and dump / rug pull memecoin scam, creator and its close circle were the whales and then he announce it to the public saying it was a 100% safe invest.

Everyone can see how it went:
https://dexscreener.com/solana/gujxvg5edqesqsjfnyybju1w5m18mzt6yyntyfjvugkm

Many people lost a lot of money on the basis of deception, and creator will face lawsuits.

I insist that an organized raid should be disregarded and the domain block should be kept in place.

**Comment by mapx-** [comment]
@iam-py-test what's your opinion ? 
I see you were pinged here too: https://github.com/hagezi/dns-blocklists/issues/3018

thanks

**Reply by JobcenterTycoon** [reply]
I pinged in Slack already: "I know nothing about crypto currency so I doubt I will be of much help, but I'll look into it tomorrow.
Sorry."

**Reply by mapx-** [reply]
https://www.alarmadefraude.com/scam/view/775260
https://es.benzinga.com/2024/06/25/pambi-criptomoneda-dalas-review-controversia/

**Reply by iam-py-test** [reply]
I know nothing more than what has already been said here.
One of the companies listed under "Partnering With" denies any involvement (Coin98), but that has already been discussed here.
As I stated earlier, I know nothing about cryptocurrency or investment fraud, so I have no clue how to investigate this beyond what I already have done.
> I see you were pinged here too

hagezi uses a malware blocklist I maintain in their Threat Intelligence Feeds blocklist, so they notify me (and a few other people) whenever a new malicious domain is reported.
Sorry

**Reply by JND94** [reply]
There are more companies that denied any involvement: https://x.com/nitrolauncher/status/1805299393563869571
One of the recent scammer posts got community noted with some info: https://twitter.com/DalasReview/status/1806843937220399365

Let's remember that this domain was flagged because it was used by twitter BOTS lying to scam people and yet somehow you fell for the most obvious malicious raid made by recently created accounts.

**Reply by stephenhawk8054** [reply]
Scam category needs another dedicated list with dedicated maintainers understanding of these fields, especially about freud or cryptocurrency. We cannot cover all of the threats without credible articles and analysis.

Badware list mainly involves:

1. Malware / Malvertising threats,
2. Phishing / Fake sites,
3. Other threats that have citations from credible analysis

We don't just put the domain to the list and leave it there. Except the obvious ones that fall into category 1 and 2 above, for categorary 3 we have to put the analysis / report source before that domain to back up the decisions: https://github.com/uBlockOrigin/uAssets/blob/master/filters/badware.txt . It's not related to raid or not. The main question for that list will always be: 

- What would be our arguments when the site owners come to complain?

The arguments cannot be just our words, it needs to have analysis from other credible sources as well.

**Reply by shootlazer** [reply]
Something I see here is that the domain is being spread all over social media while it contains brands that did not give any authorization to have their logo there.
At least, it's misleading advertising (let alone having other risks associated to cryptocurrency)

**Reply by stephenhawk8054** [reply]
Yes, misleading advertising needs to be solved by the related sides. For the threats / risks, we need sources that conduct analysis on that site.

For example, the recent addition to the list looks like this:

https://github.com/uBlockOrigin/uAssets/blob/67e6301f03824e37b8735a8c2305f3c439b5a235/filters/badware.txt#L4256-L4263

We needed to search for multiple analysis to back up the report of that site. It's not just based on the reporter.
